sql/moz-fx-data-shared-prod/mozilla_vpn_derived/all_subscriptions_v1/query.sql

```diff
@@ -376,6 +423,3 @@ SELECT
   ) AS current_months_since_subscription_start,
 FROM
   vpn_subscriptions_with_end_date
-WHERE
-  -- exclude subscriptions that never left the trial period
-  DATE(subscription_start_date) < DATE(end_date)
```

Comment:
Removing this allows trials that didn't convert to paid subscriptions, as well as subscriptions that started and ended on the same day.